### ISSUE TITLE
Save flight mission in ULOG file and ability to export it 

### ISSUE THREAD

**Issue by moreba1** [PX4/PX4-Autopilot#12538]
Hi

In a professional review of flight for  UAV development or for a crash investigation there is need to know what is total mission and ability to plot it with flight path .

**Comment by moreba1** [comment]
https://github.com/PX4/flight_review/issues/165

**Comment by Antiheavy** [comment]
agreed.  This would be a great feature to add to the ULOG.  how will you handle multiple mission uploads, for example in the middle of a flight?

**Comment by moreba1** [comment]
all of mission must be saved with time of uploading to autopilot

**Comment by julianoes** [comment]
@moreba1 pull requests are welcome :)

**Comment by junwoo091400** [comment]
This was also discussed in discord server as well: https://discord.com/channels/1022170275984457759/1032055573753118742/1034513943013638144

**Comment by junwoo091400** [comment]
Another user requested this again: https://discord.com/channels/1022170275984457759/1022170275984457762/1047648093832757269

**Comment by julianoes** [comment]
Would be nice though.
